### PR TITLE
Added ability to use extra information in the tooltip

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: stringResources/en-US/resources.resjson [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	"Role_Duration": "Duration",
 	"Role_Completion": "% Completion",
 	"Role_Resource": "Resource",
+  "Role_ExtraInformation": "Extra information"
```
Comment (uve) [inline stringResources/en-US/resources.resjson]: Please add more spaces

File: test/visualData.ts [hunk 2]
```diff
@@ -78,6 +79,17 @@ module powerbi.extensibility.visual.test {
         public valuesStartDate = GanttData.getRandomUniqueDates(this.valuesTaskTypeResource.length, new Date(2015, 7, 0), new Date(2017, 7, 0));
         public valuesDuration = GanttData.getRandomUniqueNumbers(this.valuesTaskTypeResource.length, 3, 40);
         public valuesCompletePrecntege = GanttData.getRandomUniqueNumbers(this.valuesTaskTypeResource.length);
+        public valuesExtraInformation = GanttData.getTexts(this.valuesTaskTypeResource, "Description");
+
+        public static getTexts(valuesTaskTypeResource: string[][], text: string): string[] {
+            let result: string[] = [];
+            const count: number = valuesTaskTypeResource.length;
+            for (let i: number = 0; i < count; i++) {
```
Comment (uve) [inline test/visualData.ts]: can we use **.map** function instead of for loop?

Comment (savvinsergey) [reply test/visualData.ts]: Yes, sure

Comment (savvinsergey) [reply test/visualData.ts]: Done

File: stringResources/en-US/resources.resjson (hunk 1 above)
Comment (savvinsergey) [reply stringResources/en-US/resources.resjson]: Done

**Pull request merged**